src/core/appgateway.tf

```diff
@@ -237,6 +237,23 @@ module "app_gw" {
       }
     }
 
+    api-web-io-pagopa-it = {
```

Comment:
you need to add also the rewrite rule and the backend pool for api-web

Reply:
Add route and rewrite rule in [2b9362c](https://github.com/pagopa/io-infra/pull/655/commits/2b9362c6b66967fbfaf66e7a2b35ce7445014f9c).

I didn't add a new backend pool, since we use apim as backend. Is it ok?